packages/wrangler/src/pages/build-env.ts

```diff
@@ -34,24 +38,32 @@ export const Handler = async (args: PagesBuildEnvArgs) => {
 	}
 
 	const configPath = path.resolve(args.projectDir, "wrangler.toml");
-	// Fail early if the config file doesn't exist
 	if (!existsSync(configPath)) {
-		throw new FatalError(
-			"No Pages config file found",
-			EXIT_CODE_NO_CONFIG_FOUND
-		);
+		logger.log("No Pages configuration file found. Exiting.");
+		process.exitCode = EXIT_CODE_NO_CONFIG_FOUND;
+		return;
 	}
+
 	logger.log("Reading build configuration from your wrangler.toml file...");
 
-	const config = readConfig(
-		configPath,
-		{
-			...args,
-			// eslint-disable-next-line turbo/no-undeclared-env-vars
-			env: process.env.PAGES_ENVIRONMENT,
-		},
-		true
-	);
+	let config: Omit<Config, "pages_build_output_dir"> & {
+		pages_build_output_dir: string;
+	};
+	try {
+		config = readConfig(
+			configPath,
+			{
+				...args,
+				// eslint-disable-next-line turbo/no-undeclared-env-vars
+				env: process.env.PAGES_ENVIRONMENT,
+			},
+			true
+		);
+	} catch (err) {
+		logger.log("Invalid Pages configuration file found. Exiting.");
```

Comment:
This should be a warning using logger.warn which will write to stderr instead of stdout

Reply:
This shouldn't be a warning, which will show up as a scary messaging, when this is an expected case. This should be a debug log

Reply:
I think this underlines the need for the text of these messages to be less scary themselves.

Reply:
I went ahead and done the following wrt messaging:
1. introduced a one liner `BETA` log (`Checking for configuration in a wrangler.toml (BETA)`) as per our conversation. Keeping in wrangler so we can remove more easily when time comes
2. used `logger.debug` instead of `logger.log` in the wrangler command. This gets rid of the double messaging we were seeing in the CI logs. Makes running the `build-env` command in the terminal a bit unfriendly if not run with `WRANGLER_LOG="debug"`. However, since this is not a public API, I'm OK with the tradeoff
3. Updated the CI logs as per our conversation

I'll update description with how the final messaging looks like in CI, once all things are built

Reply:
I assume the (BETA) message is in pages-infra?

Reply:
I kept the (BETA) message in wrangler, so it stays within our scope, and can remove more easily when time comes